.github/workflows/pipeline.yml

```diff
@@ -17,30 +18,28 @@ jobs:
     - name: Install dependencies
       run: |
         pip install -e .
-        pip install flake8 pytest pytest-cov codecov wheel
+        pip install -r requirements-dev.txt
     - name: Lint with flake8
       run: |
         # stop the build if there are Python syntax errors or undefined names
         flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
         # exit-zero treats all errors as warnings
         flake8 . --count --exit-zero --statistics
-    - name: Test with pytest
+    - name: Test without PyMC
       run: |
         pytest --cov=./murefi --cov-append --cov-report xml --cov-report term-missing murefi/tests.py
```

Comment:
Don't you repeat this three times now for the different pymc-versions in the matrix?